README.md

```diff
@@ -69,8 +79,18 @@ The following values are returned in `results.csv`:
 
 #### Mail sending
 
-* `MX Record` - If an MX record was found that contains at least a single mail server.
+* `MX Record` - If an MX record was found that contains at least a
+  single mail server.
 * `Mail Servers` - The list of hosts found in the MX record.
+* `Mail Server Ports Tested` - A list of the ports tested for SMTP and
+  STARTTLS support.
+* `Mail Server Is Listening` - If the mail servers are actually
```

Comment:
What does `Mail Server Is Listening` provide that `Mail Server Supports SMTP` doesn't?

Reply:
In order to try to speak SMTP to a host on a given port I have to first connect via TCP.  If that succeeds then I know _something_ is listening at that host on that port, whether it's smtp, telnet, ssh, or something else.  Since I get that information for free I was including it, but I could just as easily not record it.

I can imagine cases where the information is useful, but it may be more confusing to leave it in. 
 In any event, the same information is duplicated (and expanded upon) in the CYHY report.

Reply:
We change our mind, but I agree dropping it makes sense for now.